pod/Encoder/README.md

```diff
@@ -0,0 +1,23 @@
+# Pod Encoder
+
+Simple example for Daisy Pod showing Encoder use.
+
+Turning the Encoder cycles through which LED/color is active.
+
+Clicking the Encoder resets to LED 1 Red (on press).
+
+# Code snippet
+
+    // Debounce the Encoder at a steady, fixed rate.
+    hw.Encoder.Debounce();
+    inc = hw.Encoder.Increment();
+    // Change the selected LED based on the increment.
+    if (inc > 0)
+    {
+        led_sel += 1;
+        // Wrap around
+        if (led_sel > NUM_COLORS - 1)
+        {
+            led_sel = 0;
+        }
+    }
```

Comment:
For all code snippets you can use:

This tag to start: \```C++
And just this to end: ```

instead of doing the block quote. That way you get typical code coloring for literals, and keywords.

Might as well go through all of them like this for now.